wheels/build/wheels.yml

```diff
@@ -62,6 +62,20 @@ packages:
         prebuild:
             wheel: for f in sqlite3.h sqlite3.c sqlite3ext.h; do cp ${SRC_ROOT_1}/$f $f || return 1; done && echo 'include_dirs=.' >>${SRC_ROOT_0}/setup.cfg
         build_args: build_static bdist_wheel
+    python-ldap:
+        version: 2.4.27
+        src:
+            - http://mirror.eu.oneandone.net/software/openldap/openldap-release/openldap-2.4.44.tgz
```

Comment:
Ops! I found the http version, it's http://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.4.44.tgz